src/modules/notifications/queues/smtp.queue.ts

```diff
@@ -0,0 +1,5 @@
+export const SMTP_QUEUE = 'smtp';
```

Comment:
I see we are using mostly constants here.

But the file name smtp.queue.ts is that name good enough?

Reply:
Its more inline with Queue configuration we are passing to BullModule in NotificationModule file. So I kept it like that. That way we will also have modularity with regards to different configuration files `mailgun.queue.ts`, `sms.queue.ts`, `whatsapp.queue.ts`